kpi/tests/api/v2/test_api_submissions.py

```diff
@@ -1584,8 +1595,6 @@ def test_edit_submission_with_xml_missing_uuids(self):
         submission_xml_root = fromstring_preserve_root_xmlns(submission_xml)
         assert submission_json['_id'] == submission['_id']
         assert submission_xml_root.find('./find_this').text == 'hello!'
-        assert submission_xml_root.find('./meta/instanceID') is None
-        assert submission_xml_root.find('./formhub/uuid') is None
```

Comment:
We are rejecting submissions without `meta/instanceID` but the assertion about `formub/uuid` should be still there I think, no?